example/spec/helpers/granuleUtils.js

```diff
@@ -88,45 +90,57 @@ function createGranuleFiles(granuleFiles, bucket, oldGranuleId, newGranuleId) {
  * @returns {Promise<Object>} - input payload as a JS object with the updated granule ids
  */
 async function setupTestGranuleForIngest(bucket, inputPayloadJson, granuleRegex, testSuffix = '', testDataFolder = null) {
-  // granule id for the new files
-  const newGranuleId = randomStringFromRegex(granuleRegex);
-  console.log(`\ngranule id: ${newGranuleId}`);
+  const payloadJson = testDataFolder ?
```

Comment:
This change wasn't done with terseness in mind, it was to prevent reassigning a function parameter. It could be refactored to this if you would prefer:

```js
let payloadJson;
if (testDataFolder) {
  payloadJson = globalReplaceC('cumulus-test-data/pdrs', testDataFolder, inputPayloadJson);
} else {
  payloadJson = inputPayloadJson;
}
```

Reply:
Sorry, I had the above a bit off.  Edited.   In either case, more for your consideration than a legit PR stopping concern.